### DISCUSSION TITLE
Get a list of all running workflow instances

### DISCUSSION THREAD

**Discussion by scott-ford** [elsa-workflows/elsa-core#4550] [Q&A] [answered]
I am needing to get a list of all running workflow instances.  I believe I need to use the IWorkflowInstanceStore and call FindManyAsync.  It looks like I need to pass in a Specification.  I was wondering if I can just get the instances that are running.

**Comment by sfmskywalker** [comment, marked as answer]
You can use a specification for that. For example:

```csharp
var specification = Specification<WorkflowInstance>.Identity.WithStatus(WorkflowStatus.Running);
var workflowInstances = await _workflowInstanceStore.FindManyAsync(specification);
```
